src/libraries/System.Drawing.Common/ref/System.Drawing.Common.netcoreapp.cs

```diff
@@ -20,8 +20,10 @@ public sealed partial class Graphics
         [System.ObsoleteAttribute("Use the Graphics.GetContextInfo overloads that accept arguments for better performance and fewer allocations.", DiagnosticId = "SYSLIB0016", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
         public object GetContextInfo() { throw null; }
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Looking at the proposal that created it (https://github.com/dotnet/runtime/issues/47880), it was acknowledged that the `object`-returning variant was already EB(Never) (was true in .NET Framework), but there's no comment that these should be.

Since the ref already shipped with them visible, and the point seems to have been to make them usable... I think I'm inclined to agree that we should instead remove the attributes from the src files.

@JeremyKuhne am I missing something?

Reply:
ping @dotnet/area-system-drawing @JeremyKuhne

Reply:
Sorry, was out. I'd rather them not be browsable as they are really niche. They are really just there for Windows Forms. Almost nobody else would want to use this functionality (replicate part of the transform when drawing directly to the DC pulled from the Graphics object). It's just going to be confusing to your typical consumer here.

That said, I'm not dead set against making them browsable. I just believe it is a net negative.

Reply:
Thanks for looking, @JeremyKuhne! I've resolved this conversation then.